MCDataProducts/inc/SimParticle.hh

```diff
@@ -190,9 +190,11 @@ namespace mu2e {
     // Information at the start of the track.
     CLHEP::Hep3Vector startPosition()       const { return Geom::Hep3Vec(_startPosition);}
     CLHEP::HepLorentzVector startMomentum() const { return Geom::HepLorentzVec(_startMomentum);}
-    double      startGlobalTime()  const { return _startGlobalTime;}
-    double&     startGlobalTime()        { return _startGlobalTime;}
-    double      startProperTime()  const { return _startProperTime;}
+    XYZVectorD& startPosXYZ() { return _startPosition;}
+    XYZTVectorF& startMomXYZT() { return _startMomentum;}
```

Comment:
Like for endPos/endMom,   startPosXYZ() and startMomXYZT()  should be const functions returning const references.